zerver/lib/actions.py

```diff
@@ -817,6 +873,7 @@ def check_send_message(*args, **kwargs):
     return do_send_messages([message])[0]
```

Comment:
This one seems to be missing a type annotation?  You could detect this using `mypy --disallow-untyped-defs zerver/lib/actions.py`